test/babel-eslint.js

```diff
@@ -539,31 +498,4 @@ describe("Public API", () => {
       babelEslint.parseNoPatch("foo", {})
     );
   });
-
-  /*
-   * This test ensures that the enhanced referencer does not get used if eslint-scope has already been
-   * monkeypatched, because this causes some correctness issues. For example, if the enhanced referencer
-   * is used after the original referencer is monkeypatched, type annotation references are counted twice.
-   */
-  it("does not visit type annotations multiple times after monkeypatching and calling parseForESLint()", () => {
```

Comment:
I moved this test to `test/z_parser-for-eslint-after-patched.js` in order to not call `babelEslint.parse` before other tests.